crates/op-rbuilder/src/builders/flashblocks/payload.rs

```diff
@@ -664,7 +664,7 @@ where
             block_hash,
             transactions: new_transactions_encoded,
             withdrawals: ctx.withdrawals().cloned().unwrap_or_default().to_vec(),
-            withdrawals_root,
```

Comment:
I don't think there is correct default for this
Probably better to just expect, because it would mean that we don't support pre-isthmus chains